### DISCUSSION TITLE
gpu與cpu的比較

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#600] [Show and tell]
https://www.intel.com.tw/content/www/tw/zh/products/docs/processors/cpu-vs-gpu.html
前陣子挖礦很流行的時候，cpu也開始展現他的重要性，但我始終不理解到底cpu 與gpu 的差異，找到這篇文也才對這兩個東西開始有了些了解。整體來說，gpu更適合在遊戲上製作3d類型上的，cpu 則是在複雜運算下能發揮更大效用。

**Comment by JSHT** [comment]
GPU 的概念是 Nvidia 在 1999 年首先提出的。
對於 GPU 的產品發展有興趣的話可以看極客灣的介紹
[20年前的游戏与PC：探秘3D游戏史上最辉煌的岁月（上）](https://youtu.be/TeJ2IFae2-0)
[N卡是如何崛起的？20年前的PC和游戏（下）](https://youtu.be/JsVfKeJKJu0)

**Comment by ghost** [comment]
電獺少女也有做過相關的文章介紹 也可以去參考看看 剛入門的時候真的都會對這些很不了解@@

**Comment by handsomeniko** [comment]
GPU是組電腦最燒錢的

**Comment by bloodtrail0511** [comment]
希望RTX40系列出來之後不要又被炒價QQ

**Reply by Ben6uo** [reply]
以太轉型 應該沒30當時那麼兇吧

**Reply by JSHT** [reply]
我只擔心功耗會突破天際，電供隨便都要上 1000W 以上

**Comment by ThomasMeng0515** [comment]
GPU對於訓練機器學習也非常重要，若有強大的GPU，在作model fit的時候就不會花太久的時間

**Comment by Sazmafuji** [comment]
CPUGPU都好貴

**Reply by Ben6uo** [reply]
CPU跟GPU的價格不是同一個等級哎

**Comment by yucheng1028** [comment]
1060好像快扛不住了，我好想換3060

**Comment by Rosancl** [comment]
很久以前第一次想組電腦也研究這些東西很久

**Comment by cordydeng** [comment]
原來兩者的差別是這樣阿，長知識了~~

**Comment by minniewei** [comment]
我只知道gpu負責影像處理但從來不了解兩者結構上的差別,感謝大大分享

**Comment by gnsJhenJie** [comment]
[Mythbusters Demo GPU versus CPU - YouTube](https://www.youtube.com/watch?v=-P28LKWTzrI)
這個Nvidia拍的影片用有趣的方式解釋GPU和CPU的差別，還蠻有趣又淺顯易懂的。

**Comment by jason79461385** [comment]
如果要買GPU真的不要買AMD的，超常出錯。
